run_dir/design/project_samples.html

```diff
@@ -27,33 +27,34 @@ <h1>
   <div class="col-md-4 borderRight">
     <h4>Key Project Information</h4>
     <dl class="dl-horizontal">
-      <dt>Type</dt>									    <dd id="type">-</dd>
-      <dt>Application</dt>					    <dd id="application">-</dd>
-      <dt>Sequencing Platform</dt>	    <dd><span id="sequencing_platform">-</span></dd>
-      <dt>Sequencing Setup</dt>			    <dd><span id="sequencing_setup">-</span></dd>
-      <dt>Flowcell</dt>			            <dd><span id="flowcell">-</span></dd>
-      <dt>Flowcell Option</dt>			    <dd><span id="flowcell_option">-</span></dd>
-      <dt>Library Method</dt>				    <dd id="library_construction_method">-</dd>
-      <dt>Library Prep Option</dt>	    <dd id="library_prep_option">-</dd>
-      <dt>BioAnalyzer Run Required</dt>	<dd id="bioanalyzer_run_required">-</dd>
-      <dt>SC prep option (Hashing)</dt> <dd><span id="library_prep_option_single_cell_(hashing)">-</span></dd>
-      <dt>SC prep option (CITE)</dt>    <dd><span id="library_prep_option_single_cell_(cite)">-</span></dd>
-      <dt>SC prep option (VDJ)</dt>     <dd><span id="library_prep_option_single_cell_(vdj)">-</span></dd>
-      <dt>SC prep option (feature)</dt> <dd><span id="library_prep_option_single_cell_(feature)">-</span></dd>
```

Comment:
We don't need `feature` anymore then?

Reply:
Good catch! Yes it should have been kept.